sdk/storage/examples/list_blobs_01.rs

```diff
@@ -1,5 +1,5 @@
 use azure_core::prelude::*;
-use azure_storage::blob::prelude::*;
+use azure_storage::blob_storage::prelude::*;
```

Comment:
What was the reasoning behind adding the `storage` suffix to all the submodules?

Reply:
I've explained below, it's for the module inception: there is a blob module inside the blob module. Now we have the blob module inside the blob_storage module (which I think makes sense)